source/payment_flow_overview/index.html.md.erb

```diff
@@ -64,17 +64,15 @@ For example:
 + **description**: a human-readable description of the payment. This will be shown to the end user on the payment pages and to your staff on the GOV.UK Pay admin site (maximum 255 characters, and must not contain URLs)
 + **return_url**: an HTTPS URL on your site that the user will be sent back to once they have completed their payment attempt on GOV.UK Pay (this should not be JSON-encoded as backslashes will not be accepted)
 
-The **Create new payment** API call that the service will receive will be of the form:
-
-Example request:
+In the example, the **Create new payment** API call the service receives would give the following response header:
```

Comment:
The service isn’t receiving the **Create new payment** API call, it’s receiving the _response_ to the API call. I think something like this would do it:

“In the example, the response to the **Create new payment** API call would have headers like the following:”

(Using “like” because it’s not exact — replaying the exact example would give a different ID in the `Location` header each time)

Reply:
Makes sense, thanks 👍 : do we really need the response header or just the body?

Reply:
The fact that it’s a 201 status code and that the `Location` header is a self-link to the payment is quite useful to call out.